packages/core/src/Transaction/WithHash.ts

```diff
@@ -0,0 +1,3 @@
+import OgmiosSchema from '@cardano-ogmios/schema';
+
+export type WithHash = { hash: OgmiosSchema.Hash16 } & OgmiosSchema.Tx;
```

Comment:
I used different style for exporting types for a module in `cip2`, using one `types.ts` instead of file per type. We should use one style of the other for consistency, I don't mind either. Let me know if you want me to refactor `cip2`.

Reply:
I'll conform to what we already have

Reply:
Actually, will defer to another PR as there's others to align